src/backend/tests/c-lisp/iteration.sexp

```diff
@@ -0,0 +1,18 @@
+(c-lisp
+    (define ((print int) (n int)))
+
+    (define ((main int))
+        (declare (i int))
+
+        (for (set i 11)
```

Comment:
Use this syntax (for (init, cond, step) body)

Reply:
Done